disvae/losses.py

```diff
@@ -204,27 +212,27 @@ def __call__(self, data, model, optimizer, is_train, storer):
         batch_size = data.size(dim=0)
         half_batch_size = batch_size // 2
         data = data.split(half_batch_size)
-        data1 = data[0].to(self.device)
-        data2 = data[1].to(self.device)
+        data1 = data[0]
+        data2 = data[1]
```

Comment:
Quick opinion @YannDubs and @alecokas. Currently, `data1` and `data2` will be the same every epoch. Wondering if I should switch them around each epoch?

Reply:
I'm not sure. Let me get back to you.

Reply:
I don't think you need to: the data is being shuffled during training in any case